src/Settings.cpp

```diff
@@ -116,14 +111,14 @@ ADDON_STATUS Settings::ReadBackendSettings()
       if (XMLUtils::GetInt(settingsDoc.RootElement(), "NextPVRVersion", m_backendVersion))
       {
         // NextPVR server
-        XBMC->Log(LOG_INFO, "NextPVR version: %d", m_backendVersion);
+        kodi::Log(ADDON_LOG_INFO, "NextPVR version: %d", m_backendVersion);
 
         // is the server new enough
         if (m_backendVersion < 40204)
         {
-          XBMC->Log(LOG_ERROR, "Your NextPVR version '%d' is too old. Please upgrade to '%s' or higher!", m_backendVersion, NEXTPVRC_MIN_VERSION_STRING);
-          XBMC->QueueNotification(QUEUE_ERROR, XBMC->GetLocalizedString(30050));
-          XBMC->QueueNotification(QUEUE_ERROR, XBMC->GetLocalizedString(30051), NEXTPVRC_MIN_VERSION_STRING);
+          kodi::Log(ADDON_LOG_ERROR, "Your NextPVR version '%d' is too old. Please upgrade to '%s' or higher!", m_backendVersion, NEXTPVRC_MIN_VERSION_STRING);
+          kodi::QueueNotification(QUEUE_ERROR, "Change - Need Header",  kodi::GetLocalizedString(30050));
+          //kodi::QueueNotification(QUEUE_ERROR, kodi::GetLocalizedString(30051), NEXTPVRC_MIN_VERSION_STRING);
```

Comment:
About things like this gives it a QueueFormatedNotification, see http://alwinesch.github.io/group__cpp__kodi.html#ga39c65205ea763e00c402cb0f1f50b289

Reply:
It turns out that is how I resolved it.

```
kodi::QueueNotification(QUEUE_ERROR, kodi::GetLocalizedString(30050), StringUtils::Format(kodi::GetLocalizedString(30051).c_str(), NEXTPVRC_MIN_VERSION_STRING));